tests/extendrtests/src/rust/src/attributes.rs

```diff
@@ -0,0 +1,42 @@
+use extendr_api::prelude::*;
+
+#[extendr]
+fn dbls_named(x: Doubles) -> Doubles {
```

Comment:
Now that the `@export` is removed, the `cargo extendr document` needs to be called to update wrappers.
I'm pushing a commit that does this now.